bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZGTo4L_4f_NLO/ZZGTO4L_4f_NLO_run_card.dat

```diff
@@ -0,0 +1,144 @@
+#***********************************************************************
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+ .true.   = reweight_scale   ! reweight to get scale dependence
+  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+#***********************************************************************
+# Merging - WARNING! Applies merging only at the hard-event level.     *
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 0        = ickkw            ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+ 30  = bwcutoff
+#***********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+   1  = jetalgo   ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+ 0.7  = jetradius ! The radius parameter for the jet algorithm
+  10  = ptj       ! Min jet transverse momentum
+  -1  = etaj      ! Max jet abs(pseudo-rap) (a value .lt.0 means no cut)
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+   3  = ptl     ! Min lepton transverse momentum
+  -1  = etal    ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
+   0  = drll    ! Min distance between opposite sign lepton pairs
+   0  = drll_sf ! Min distance between opp. sign same-flavor lepton pairs
+   1  = mll     ! Min inv. mass of all opposite sign lepton pairs
+   1  = mll_sf  ! Min inv. mass of all opp. sign same-flavor lepton pairs
```

Comment:
You would get some divergence from gamma*->l+ l- with this 1 GeV mass cut. Is this what you really want for your analysis? If you are going to apply some mass cuts on mll near Z mass pole in RECO level, suggest to move it up to 50

Reply:
for z2, we have primary selection between 4~120 GeV, maybe we can chage mll_sf=4.

Reply:
Then wouldn't it be better to use MadSpin by doing
generate p p > ell+ ell- z a
decay z > ell+ ell-
if you require mZ1 ~ 90 GeV?

To secure/claim more stats for your samples

Reply:
And the computations would be faster

Reply:
follow the advice from Olivier [here](https://answers.launchpad.net/mg5amcnlo/+question/696360), The symmetry factor can not be set in a consistent way in process p p > z l l a.

Reply:
Ah never knew this thanks for the clarification